prometheus-exporters/esxi-host-exporter/templates/deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             - mountPath: /atlas
               name: atlas-output
               readOnly: true
-            - mountPath: /
+            - mountPath: /exporter/
```

Comment:
```suggestion
            - mountPath: /exporter/esxi-exporter-config.yaml
```

Reply:
why is this necessary?

Reply:
It is indeed not needed, just make sure you are using `/exporter/config.yaml` as the file to read in the exporter.

Reply:
yes thats the case:
https://github.com/sapcc/esxi-exporter/blob/main/modules/configuration/ExporterConfig.py#L24

Reply:
well, ok ^^